src/types/gameConsole.ts

```diff
@@ -57,6 +57,12 @@ interface OutputTokens {
   // TWiLightMenu++ Roms go into the /roms subfolder on the 3DS/DSi SD card
   // @see https://github.com/DS-Homebrew/TWiLightMenu/tree/master/7zfile/roms
   twmenu?: string,
+  
+  // Console brand, name and abbrevited name
+  // @see https://wiki.no-intro.org/index.php?title=Systems
+  consoleBrand?: string,
```

Comment:
This token makes sense to me, but I think it's only valuable if it's required for all `GameConsole`s:

```suggestion
  consoleBrand: string,
```

Reply:
Can't agree more !